CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 <!-- For changes in existing functionality.  -->
 - Autoscaling is now directly managed inside Kubernetes, by the Constellation node operator.
+- The `constellation create` on GCP now uses Terraform to create and destroy cloud resources.
```

Comment:
You can also add the GCP no public IP change here (and that SSH now needs to be manually allowed before it can be used).